automation/snyk/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM ${OPENVINO_UBUNTU_IMAGE}
+
+ARG http_proxy
```

Comment:
I believe, we can remove proxy args

Reply:
I think it is needed later in the ENV for the Snyk.